packages/toggle/stories/Toggle.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { withReadme } from "storybook-readme";
+import { Toggle } from "../../index";
+
+const readme = require("../README.md");
+
+const primary = () => <div>primary component</div>;
+const secondary = () => <div>secondary component</div>;
+
+storiesOf("Toggle", module)
+  .addDecorator(withReadme([readme]))
+  .addWithInfo("string", () => (
+    <Toggle primaryContent="Hello" secondaryContent="Bye" />
```

Comment:
@mperrotti I have addressed the ARIA/role issue which discards the second code suggestion.

I was advocating for us to abstract the logic and component entirely, meaning the developer only need to pass the data and UI Kit takes care of displaying it.

I'm open to leaving a bit more flexible using the render prop interface but I would rather see us not giving that condition flexibility in this component.

Reply:
What would you suggest for the props to replace `primaryContent` and `secondaryContent`

Reply:
If `primary-` and `secondary-` are a familiar naming convention for toggle content/values, we should keep what you have instead of making people learn something new

Reply:
What do you think about `on` `off`?

Reply:
What about `contentOn` and `contentOff`? I think adding `content-`makes it clearer that those props accept content to be rendered. `on` and `off` seem a little mysterious to me

Reply:
Sounds good.